cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -32,41 +32,45 @@
 {% set page_url = get_protected_url(ancestor) %}
 
 <div class="o-item-introduction">
-    <div class="h4">
+    {% if value.category is string %}
         {{ category_slug.render(value.category, page_url, index=form_id) }}
-    </div>
-    <h1>{{ value.heading }}</h1>
+    {% else %}
+        {% for category in page.categories.all() %}
+            {{ '|' if loop.index > 1 else '' }}
+            {{ category_slug.render(category.name, get_protected_url(ancestor), 'post_slug', index=index) }}
+        {% endfor %}
+    {% endif %}
+    <h1>{{ value.heading | safe }}</h1>
 
     {% if value.paragraph %}
         <div class="lead-paragraph">{{ parse_links(value.paragraph) | safe }}</div>
     {% endif %}
+
     {% if value.date or page.authors.all() %}
```

Comment:
should we just create a variable like `has_authors = page.authors.all()|length > 0`. It could be used below as well.